From c09a6def153e52494072a36233c7e7b3307b67bf Mon Sep 17 00:00:00 2001
From: Cassidy James Blaede <c@ssidyjam.es>
Date: Mon, 22 Apr 2024 21:38:51 -0600
Subject: [PATCH] simple-scan: Use RDNN app ID

This was preventing the GResourced icons from being used. Fixes
https://gitlab.gnome.org/GNOME/simple-scan/-/issues/390
---
 src/simple-scan.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/simple-scan.vala b/src/simple-scan.vala
index c2e2da2c..cedbe060 100644
--- a/src/simple-scan.vala
+++ b/src/simple-scan.vala
@@ -40,7 +40,7 @@ public class SimpleScan : Adw.Application
     public SimpleScan (ScanDevice? device = null)
     {
         /* The inhibit () method use this */
-        Object (application_id: "simple-scan");
+        Object (application_id: "org.gnome.SimpleScan");
         register_session = true;
 
         default_device = device;
-- 
2.39.2

